sass/components/list.sass

```diff
@@ -0,0 +1,22 @@
+$list-border: 1px solid $border !default
+$list-background-color: $white !default
+$list-radius: $radius !default
+
+$list-item-border: 1px solid $border !default
+$list-item-hover-color: $background !default
+
+.list
+  background-color: $list-background-color
+  border: $list-border
+  border-radius: $list-radius
+  .list-item
+    color: black
+    cursor: default
+    display: flex
+    justify-content: flex-start
+    padding: 8px 16px
```

Comment:
Use `padding: 0.5em 1em`

Reply:
i didn't understand em and rem until today xD